cpu/fe310/include/context_frame.h

```diff
@@ -0,0 +1,138 @@
+
+/*
+ * Copyright (C) 2017 JP Bonn
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_fe310
+ * @{
+ *
+ * @file
+ * @brief       Thread context frame stored on stack.
+ *
+ * @author      JP Bonn
+ */
+
+#ifndef CONTEXT_FRAME_H
+#define CONTEXT_FRAME_H
+
+#if !defined(__ASSEMBLER__)
+#include <stdint.h>
+#include <assert.h>
+#endif /* __ASSEMBLER__ */
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if !defined(__ASSEMBLER__)
+
+/**
+ * @brief   context_switch_frame stores the registers and PC for a context switch.
+ *
+ * This also defines context_switch_frame offsets for assembly language.  The
+ * structure is sized to maintain 16 byte stack alignment per the ABI.
+ * https://github.com/riscv/riscv-elf-psabi-doc
+ *
+ */
+struct context_switch_frame {
+    uint32_t filler0;               /**< filler to maintain 16 byte alignment */
+    uint32_t filler1;               /**< filler to maintain 16 byte alignment */
```

Comment:
I think it is more commonplace to write the padding as 
```c
uint32_t padding[2];
```
So you don't need to add more lines to adjust the padding.

Reply:
Fixed